docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+# For locally developing the add-on as a standalone Docker container 
+version: '3.8'
+
+services:
+  asterisk:
+    image: ghcr.io/tech7fox/amd64-addon-asterisk
+    build:
+      context: ./asterisk
+      network: host
+    tty: true
+    entrypoint: /init
+    init: false
+    environment:
+      # To connect to the HA API
+      HOME_ASSISTANT_URL: http://home-assistant:8123
+      HOME_ASSISTANT_TOKEN: <my_secret_token>
+    volumes:
+      # Mount configuration read by the addon
+      - ./asterisk/config.json:/data/options.json:ro
+      
+      # To avoid having to rebuild the whole image, mount the init files inside it
+      # - ./asterisk/rootfs/etc/cont-init.d/asterisk.sh:/etc/cont-init.d/asterisk.sh:ro
+      # - ./asterisk/rootfs/etc/cont-init.d/asterisk_mbox.sh:/etc/cont-init.d/asterisk_mbox.sh:ro
```

Comment:
If this is causing the build to take long, probably we need to fix it. Usually docker build has to be fast if it's just some files changed.